src/Icon/Icon.js

```diff
@@ -34,7 +34,7 @@ Icon.propTypes = {
     /** CSS class(es) to add to the element */
     className: PropTypes.string,
     /** Size of the component: 's', 'm', 'l', 'xl' */
-    size: PropTypes.oneOf(ICON_SIZES)
+    size: PropTypes.oneOf(Object.keys(ICON_SIZES))
```

Comment:
let's choose a default size

Reply:
Actually let's not so it just gets the whatever font-size - can we make that prop conditional so we don't have undefined when it's not specified though

Reply:
Look at the diff for InputGroup:
```
<span
          className="sap-icon--globe"
          style={
            Object {
              "fontSize": undefined,
            }
          }
        />
```
It should probably not even add the style prop if fontSize is undefined instead of adding that